File: examples/tree-sitter-test/php/hello.php
<?php

class Greeter {
    private string $name;

    public function __construct(string $name) {
        $this->name = $name;
    }

    public function greet(): string {
        return "Hello, {$this->name}!";
    }
}

$g = new Greeter("World");
echo $g->greet() . "\n";

File: examples/tree-sitter-test/go/hello.go
package main

import "fmt"

type Greeter struct {
	Name string
}

func (g Greeter) Greet() string {
	return fmt.Sprintf("Hello, %s!", g.Name)
}

func main() {
	g := Greeter{Name: "World"}
	fmt.Println(g.Greet())
}
